_data/errors.yml

```diff
@@ -83,11 +83,39 @@ oidc:
         - Ensure the `redirect_uri` that you are sending as part of your authentication request is registered. You must register every redirect uri that your application redirects through.
     - title: Unauthorized scope
       id: oidc-unauthotized-scope
-      accordion-id: oidc-unauth-scope
+      accordion-id: oidc_unauth_scope
       content: |
         ##### Why it's happening
         You have requested attributes that your Service Level does not allow.
         ##### What to do:
         - Check the `scope` parameter in your authentication request.
         - If your application is an *Authentication Only* application, ensure you are only passing in IAL1 attributes.
         More information about the different types of attributes can be found here: [https://developers.login.gov/attributes/](https://developers.login.gov/attributes/){:target="_blank"}
+  logout:
+    - title: client_id was not recognized
+      id: client-id-not-recognized
+      accordion-id: client_id_not
```

Comment:
Would it still be correct to capitalize Client in the accordion title? Client in the first two titles of this section are lower case and it loses consistency.

Reply:
resolved here! https://gsa-tts.slack.com/archives/C01SU3NB9T3/p1699380161937509?thread_ts=1699373843.731929&cid=C01SU3NB9T3